.golangci.yml

```diff
@@ -126,10 +126,9 @@ linters:
         - G101
         - G305
     govet:
-      enable:
-        - fieldalignment
       disable:
         - copylocks
```

Comment:
somewhat unrelated, but why was this added?

Reply:
from https://pkg.go.dev/github.com/golangci/govet#section-readme

> Locks that are erroneously passed by value.

could you remove it from the disabled list?